frontend/packages/console-shared/src/utils/index.ts

```diff
@@ -0,0 +1,5 @@
+export const capitalizeFirstLetter = (str) => str.charAt(0).toUpperCase() + str.slice(1);
+
+export function prefixedId(idPrefix, id) {
+  return idPrefix && id ? `${idPrefix}-${id}` : null;
```

Comment:
Wouldn't it be better to use whatever is available instead of skipping at all?

Reply:
no, because this should be html id and the ids could become duplicated if the prefix or id is missing

Reply:
Side note: in cases where we just need some unique ID, we can always use `_.uniqueId` function which has an optional `prefix` parameter.